spec.bs

```diff
@@ -681,6 +688,9 @@ maps to an internal [=fenced frame config=] [=struct=].
 
 Issue: Fix the "indistinguishable" IDL bug with the unions above.</span>
 
+Each {{FencedFrameConfig}} has a <dfn for=fencedframeconfig>url</dfn>, which is a [=URL=] or null,
```

Comment:
The reason why I deleted this is: what does it mean for `url` to be "initially null"? All {{FencedFrameConfig}} constructors should specify the value of `url` (which might be null, but it's not like that's a default).

Reply:
Oh, you added this back to fix the compile warning...

Reply:
That's right. Maybe I'll just make it a URL that's not initially null, and that has to be provided upon construction.